longestCommonSubSequence.js

```diff
@@ -132,9 +146,78 @@ function lcSubStrTD(x, y, n, m){
     return result;
 }
 
+function printLCS(x, y, n, m, result= ''){
```

Comment:
This method seems to have some issues, doesn't pass all the test cases

Reply:
Yeah that method isn't proper, doesn't pass all the test cases. I would like you to look at printLCSTD which i hope does pass all the test cases.